scripts/tests.unit.sh

```diff
@@ -17,4 +17,4 @@ source "$HYPERSDK_PATH"/scripts/constants.sh
 
 # Provision of the list of tests requires word splitting, so disable the shellcheck
 # shellcheck disable=SC2046
-go test -race -timeout="3m" -coverprofile="coverage.out" -covermode="atomic" $(find . -name "*.go" | grep -v "./x/programs/cmd" | grep -v "./examples/morpheusvm" | grep -v "./examples/tokenvm" | xargs -n1 dirname | sort -u | xargs)
+go test -race -timeout="3m" -coverprofile="coverage.out" -covermode="atomic" $(find . -name "*.go" | grep -v "./x/programs/cmd" | grep -v "./x/programs/v2/cmd" | grep -v "./examples/morpheusvm" | grep -v "./examples/tokenvm" | xargs -n1 dirname | sort -u | xargs)
```

Comment:
Do we not want to run the `cmd` tests? If there are test files, we should probably run them, right? If we want to trigger all the tests from Rust, we can just not write any test-files.

Reply:
I totally forgot that `-v` was to invert the match, you're right, I will remove this